cmd/antrea-controller/controller.go

```diff
@@ -131,13 +131,14 @@ func createAPIServerConfig(kubeconfig string,
 	secureServing.ServerCert.CertDirectory = ""
 	secureServing.ServerCert.PairName = "antrea-apiserver"
 	secureServing.BindPort = bindPort
+	secureServing.BindAddress = net.ParseIP("0.0.0.0")
 	// kubeconfig file is useful when antrea-controller isn't not running as a pod, like during development.
 	if len(kubeconfig) > 0 {
 		authentication.RemoteKubeConfigFile = kubeconfig
 		authorization.RemoteKubeConfigFile = kubeconfig
 	}
 
-	if err := secureServing.MaybeDefaultWithSelfSignedCerts("localhost", nil, []net.IP{net.ParseIP("127.0.0.1")}); err != nil {
+	if err := secureServing.MaybeDefaultWithSelfSignedCerts("127.0.0.1", nil, []net.IP{net.ParseIP("127.0.0.1")}); err != nil {
```

Comment:
Why we need this change? I see all code in K8s for such case is using "localhost", though I don't know the difference.

Reply:
Sorry, this line was changed by my local experiment. I would revert this line.
BTW, if we disable `--insecure`, we need to provide the IP which outside may access by, although there are many other requirements.